enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+
+## Upgrade / Downgrade Strategy
+
+Operators implementing support for change management will carry forward their
+existing upgrade and downgrade strategies.
+
+## Version Skew Strategy
+
+Operators implementing change management for their resources will not face any
+new _internal_ version skew complexities due to this enhancement, but change management 
+does increase the odds of prolonged and larger differential kubelet version skew.
+
+For example, particularly given the Manual or Assisted change management strategy, it 
+becomes easier for a cluster lifecycle administrator to forget to update worker-nodes
+along with updates to the control-plane. 
+
+At some point, this will manifest as the kube-apiserver presenting as Upgradeable=False,
+preventing future control-plane updates. To reduce the prevalence of this outcome,
+the additional responsibilities of the cluster lifecycle administrator when 
+employing change management strategies must be clearly documented along with SOPs
+from recovering from skew issues.
+
+HyperShift does not have any integrated skew mitigation strategy in place today. HostedCluster
+and NodePool support independent release payloads being configured and a cluster lifecycle
+administrator can trivially introduce problematic skew by editing these resources. HyperShift
+documentation warns against this, but we should expect a moderate increase in the condition
+being reported on non-managed clusters (OCM can prevent this situation from arising by
+assessing telemetry for a cluster and preventing additional upgrades while worker-node
+configurations are inconsistent with the API server). 
+
+## Operational Aspects of API Extensions
+
+The API extensions proposed by this enhancement should not substantially increase
+the scope of work of operators implementing the change management support. The
+operators will interact with the same underlying resources/CRDs but with
+constraints around when changes can be initiated. As such, no significant _new_ 
+operational aspects are expected to be introduced.
+
+## Support Procedures
+
+Change management problems created by faulty implementations will need to be resolved by 
+analyzing operator logs. The operator responsible for a given resource will vary. Existing
+support tooling like must-gather should capture the information necessary to understand
+and fix issues. 
+
+Change management problems where user expectations are not being met are designed to
+be informed by the detailed `status` provided by the resources bearing the `changeManagement`
+stanza in their `spec`.
+
+## Alternatives
+
+### Implement maintenance schedules via an external control system (e.g. ACM)
+We do not have an offering in this space. ACM is presently devoted to cluster monitoring and does
+not participate in cluster lifecycle.
```

Comment:
We could add it there, though, right? Or as an OLM-installed operator?

Reply:
Yes, it could. The conclusion presented here was with guidance from Derek.